packages/create-platformatic-db/src/create-package-json.mjs

```diff
@@ -0,0 +1,33 @@
+import pupa from 'pupa'
+import { isFileAccessible } from './utils.mjs'
+import { writeFileSync } from 'node:fs'
+import { join } from 'node:path'
+
+const packageJsonTemplate = `\
+{
+  "name": "platformatic-db-api-example",
+  "version": "0.0.1",
+  "description": "Platformatic DB API",
+  "scripts": {
+    "start": "platformatic db start"
+  },
+  "devDependencies": {
+    "fastify": "^{fastifyVersion}",
+    "platformatic": "^{platVersion}"
```

Comment:
The `platformatic` package should be in `dependencies` not `devDependencies` to ensure that it's installed and available in production environments (we can't assume that users are using a Docker image where the `platformatic` CLI is available.

Reply:
On the subject of dependencies, when the `platformatic db init` command completes it outputs a message saying:

```bash
npm i --save fastify@4.10.2 @platformatic/sql-mapper@0.10.0 @platformatic/sql-graphql@0.10.0` to install types dependencies.
````

Is there a reason we don't need those `@platformatic/*` dependencies for their types here?